sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/models/LogsQueryClientAudience.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.query.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.azure.monitor.query.LogsQueryClient;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Collection;
+
+/**
+ * Cloud audiences available for {@link LogsQueryClient}.
+ */
+public final class LogsQueryClientAudience extends ExpandableStringEnum<LogsQueryClientAudience> {
```

Comment:
Why do we have separate types for logs and metrics?

Reply:
Also, do you need to set up a pipeline for the non public cloud. That can be done via a small change to the yaml file.

Reply:
Logs and metrics use two different services and there are 2 separate clients. So, the audience for each can be different.